subsys/bluetooth/services/fast_pair/fmdn/state.c

```diff
@@ -98,6 +98,9 @@ enum utp_control_flags_bit_num {
 	UTP_CONTROL_FLAGS_BIT_NUM_RING_AUTH_SKIP = 0,
 };
 
+/* Verify if the conducted Bluetooth TX power for FMDN advertisements is set to at least 0 dBm. */
+BUILD_ASSERT(CONFIG_BT_FAST_PAIR_FMDN_TX_POWER >= 0);
```

Comment:
I am not sure if we should make it a build assert or e.g. a warning only. The specification uses `should` term.

Reply:
After consultation with the specification authors, it seems like a requirement.

Reply:
Thanks for clarifying that one.